webpage/content/features/rendering.md

```diff
@@ -120,6 +120,20 @@ menu:
 <td style="text-align: center" class="ok">✓<sup>1</sup></td>
 <td style="text-align: center" class="ok">✓</td>
 </tr>
+
+<tr>
+<td style="text-align: left">GDI+</td>
+<td style="text-align: center" class="ok">✓</td>
+<td style="text-align: center" class="ok">✓</td>
+<td style="text-align: center" class="ok">✓</td>
+<td style="text-align: center" class="missing">✗</td> 
+<td style="text-align: center" class="ok">✓</td>
+<td style="text-align: center" class="ok">✓</td>
+<td style="text-align: center" class="ok">✓</td>
+<td style="text-align: center">?</td>
```

Comment:
You may check it with street names in Israel ;-) For example this one: https://www.openstreetmap.org/way/230895035#map=18/31.79142/35.22386

Reply:
Many thanks for the tip! I will try that out.